content/post/2025-04-08-elasticc/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Preparing machine learning classifiers for Rubin
+date: 2025-04-08
```

Comment:
When do you want this to appear? (also in the folder name)

Reply:
I do not know, how frequent we want these to be? once a week?

Reply:
If we plan the exoplanet on the 14/04 -- Elasticc can be 21/04?

Reply:
I think that is fine! I renamed the folder